### ISSUE TITLE
Opening the viz full screen too quickly in calculation made a blank screen appear and the value never showed

### ISSUE THREAD

**Issue by sylwiabr** [enso-org/enso#6526]
### What type of issue is this?

Transient – Occurring only once

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

https://user-images.githubusercontent.com/12892578/235895928-81dc63b5-ced2-421f-b092-1cb1b44466a5.mov

### Expected behaviour

Full screen vis should be updated without reopening

### Enso Version

Nightly 02.03.2023

### Browser or standalone distribution

Standalone distribution

### Browser Version or standalone distribution

Standalone

### Operating System

MacOS

**Comment by vitvakatu** [comment]
Seems to be a race condition in IDE:

<img width="446" alt="Screenshot 2023-05-08 at 01 23 21" src="https://user-images.githubusercontent.com/6566674/236704024-894e0e51-d69e-42e5-a9a3-cac7dc2fa815.png">

**Comment by sylwiabr** [comment]
Duplicate of more general issue #6561
